.circleci/config.yml

```diff
@@ -156,12 +157,20 @@ jobs:
       - run:
           name: Bootstrap the test applications in OpenShift
           command: |
-            LOCALE_IP=$(hostname -I | awk '{print $1}')
-            oc cluster up --server-loglevel=5 --public-hostname="${LOCALE_IP}"
-            oc login -u developer -p developer https://${LOCALE_IP}:8443 --insecure-skip-tls-verify=true
+            OPENSHIFT_DOMAIN=$(hostname -I | awk '{print $1}')
+            oc cluster up --server-loglevel=5 --public-hostname="${OPENSHIFT_DOMAIN}"
+            oc login -u developer -p developer https://${OPENSHIFT_DOMAIN}:8443 --insecure-skip-tls-verify=true
             echo "arnold" | USE_TTY=false ./bin/bootstrap -e "env_type=ci"
-            sleep 10
-            wget --no-check-certificate "https://hello.patient0-ci.${LOCALE_IP}.nip.io"
+            for try in $(seq 5); do
+              echo "Testing hello application http response ($try)"
+              curl -vk "https://next.hello.ci-patient0.${OPENSHIFT_DOMAIN}.nip.io/" 2> /tmp/hello.err > /tmp/hello.out
+              if grep "HTTP/1.1 200 OK" /tmp/hello.err && grep "Hello OpenShift! by Arnold" /tmp/hello.out; then
```

Comment:
why woul the 200 code response be in written to stderr?

Reply:
In verbose mode, `curl` displays headers, response status etc in `stderr` and the http response content in `stdout`... and it works :tada:

Reply:
:+1: